### DISCUSSION TITLE
Why doesn't mongodb support

### DISCUSSION THREAD

**Discussion by kangyunfu** [apache/superset#19589] [Q&A / Help]
Why doesn't mongodb support

**Comment by srinify** [comment]
Superset uses SQLAlchemy as an abstraction layer to talk to databases and MongoDB doesn't have a reliable SQLAlchemy dialect.

https://preset.io/blog/building-database-connector/

You can connect Superset to Presto or Trino instead and have either of those engines query MongoDB!

https://preset.io/blog/2021-6-22-trino-superset/

Moving to discussions as this isn't a bug :)
